lib/config/wrapper.go

```diff
@@ -549,3 +551,21 @@ func (w *wrapper) AddOrUpdatePendingFolder(id, label string, device protocol.Dev
 
 	panic("bug: adding pending folder for non-existing device")
 }
+
+func (w *wrapper) AddOrUpdateFolderCandidateDevice(folder string,
+	device, introducer protocol.DeviceID, name, certName string, addresses []string) {
+	defer w.Save()
```

Comment:
Copied verbatim from `AddOrUpdatePendingDevice()` and others.  Sorry I'm new to Go and have no idea how that should be checked in conjunction with `defer`.

This will not live in the config anyway, long-term.